cmd/jimmctl/cmd/migratemodel.go

```diff
@@ -20,15 +17,24 @@ import (
 
 const (
 	migrateModelCommandDoc = `
-The migrate command migrates a model(s) to a new controller. Specify
-a model-uuid to migrate and the destination controller name.
+The migrate commands migrates a model, or many models between two controllers
+registered within JIMM. 
+
+You may specify a model name (of the form owner/name) or model UUID.
+
+For example, JIMM is aware of controller-a and controller-b and controller-a has
+models:
+	- alice@canonical.com/model-a
+	- 2cb433a6-04eb-4ec4-9567-90426d20a004
```

Comment:
This is confusing, are we saying there is one model or two? And why if there's two why is the one being shown with a name and the other with a UUID?

Reply:
It's because we want to allow names AND uuids.

Reply:
Yeah but we don't need to show them with mixed usage like this. We can just say you can provide one and/or the other. This is very confusing to read.

Reply:
Done, removed and left example as suggested lower down